developers/README.md

```diff
@@ -2,75 +2,65 @@
 
 ## Table of Contents
 
-* [Development Environment](#development-environment)
-  * [Virtual Machines](#virtual-machines)
 * [Contributing](#contributing)
-* [IRC](#irc)
+* [Discussion](#discussion)
+* [Local Development Environment](#local-development-environment)
+  * [Local Testing](#local-testing)
+* [Ansible Environments](#ansible-environmnets)
```

Comment:
`Ansible Environments` comes before `Local Development Environment` in the lore. Please reverse their order in the table of contents.